api/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+  api:
+    build:
+      context: .
+    ports:
+      - "5000:5000"
+    environment:
+      - FLASK_ENV=production
+    volumes:
+      - .:/usr/src/app
+    command: flask run --host=0.0.0.0 --port=5000
```

Comment:
I am also curious if we should take another look at our pattern for deploying Flask apps w/o Lambda wrapping them. This would affect citybase, the EAZ analysis tool, this service, and Moped API in the future.

Here is the specific warning that I saw in the EAZ pr `WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.`. I also suspect that Lambda covered us here in the past.

I didn't have time to spin up the local API to see if we see the same warning but it could be good to check and make sure we aren't missing something.

Reply:
As an update, I attached to the API container, and I see the same warning.

Reply:
We should, 100%. In this case, we don't push enough traffic onto the service to care, but we really should rethink this practice. The warning is right there when you start it.

We are just waiting for someone to DDOS this endpoint into oblivion as it were. But, I do think the ECS micro instance we run this on will be what breaks way before the service, but the point still stands. I would be so excited to get an issue to sort these out and develop a deployment pattern to bring all of our flask apps up to snuff.